src/components/dialogs/movie-dialog/MovieDialog.tsx

```diff
@@ -2,12 +2,20 @@ import BaseDialog from "../base-dialog/BaseDialog";
 import {MovieDialogData} from "../../../models/movies";
 import MovieForm from "./movie-form/MovieForm";
 import React from "react";
+import {Portal} from "react-portal";
 
-function MovieDialog({onClose, onSubmitChanges, title, movie}: MovieDialogData): React.JSX.Element {
+function MovieDialog({isOpen, onClose, onSubmitChanges, title, movie}: MovieDialogData): React.JSX.Element {
     return (
-        <BaseDialog title={title} onClose={onClose}>
-            <MovieForm onClose={onClose} movie={movie} onSubmitChanges={onSubmitChanges}/>
-        </BaseDialog>
+        <>
+            {isOpen &&
```

Comment:
You could return null instead. Just write a simple if statement before the main 'return'.
if (!isOpen) { retrun null; }